Storage/src/Connection/Rest.php

```diff
@@ -113,135 +115,135 @@ public function projectId()
      */
     public function deleteAcl(array $args = [])
     {
-        return $this->send($args['type'], 'delete', $args);
+        return $this->sendRequest('bucket_acl', 'delete', $args);
```

Comment:
Can you check if the caller of `deleteAcl` is sending `type` in the `$args/$options` just for this, if so, then we can remove it from the caller.

Reply:
This function is used for `bucket_acl`, `object_acl` as well as `default_object_acl`, thus we can't hardcode it.

Reply:
Actually this is a mistake. If you see here, the type they send is 
            'bucketAccessControls',
            'defaultObjectAccessControls',
            'objectAccessControls'
I will change this to the correct way to receive this parameter.

Reply:
Thank you for noticing this

Reply:
ref: https://github.com/googleapis/google-cloud-php/blob/main/Storage/src/Acl.php#L66